contracts/src/token/erc1155/extensions/uri_storage.rs

```diff
@@ -98,12 +101,11 @@ mod tests {
         U256::from(num)
     }
 
-    sol_storage! {
-        struct Erc1155Example {
-            Erc1155 erc1155;
-            Erc1155MetadataUri metadata_uri;
-            Erc1155UriStorage uri_storage;
-        }
+    #[storage]
+    struct Erc1155Example {
+        pub erc1155: Erc1155,
```

Comment:
It seems [we don't need this field for unit tests](https://github.com/OpenZeppelin/rust-contracts-stylus/actions/runs/12272439443/job/34241193547?pr=438)

Reply:
Do you mean it is not used?

Reply:
Your link lead to the ci root. Not obvious to me wdm

Reply:
Sorry, I think it used to open the page at the exact warning line in the workflow before :thinking: 

![image](https://github.com/user-attachments/assets/f4038971-3438-471d-a420-34cfa86f8a5f)

Reply:
now I see..

Reply:
Okay. Lets drop this field then..